php/slim/public/index.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Slim\App;
+use Slim\Http;
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+
+$app = new App();
+
+$app->get('/', function(Http\Request $request, Http\Response $response): Http\Response {
+    return $response->write('');
+});
+
+$app->get('/user/{id}', function(Http\Request $request, Http\Response $response, array $args): Http\Response {
+    return $response->write($args['id']);
+});
+
+$app->get('/user', function(Http\Request $request, Http\Response $response): Http\Response {
```

Comment:
This route **SHOULD** be `POST`, but I have a `405` **Method not allowed**

I don't know how to by pass this behavior on `slim`

Reply:
sorry, I mean I have a `405` when using `post`
@see https://www.slimframework.com/docs/v3/handlers/not-allowed.html

and I do not know how to escape this (I have to do the same for `django` for example)